### DISCUSSION TITLE
LazyVim cannot be updated to the latest commit

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#4837] [Q&A] [answered]
this my config in `config/lazy.lua`, i have set version to false.

```lua
require("lazy").setup({
  spec = {
    -- add LazyVim and import its plugins
    { "LazyVim/LazyVim", import = "lazyvim.plugins" },
    { import = "plugins" },
  },
  defaults = {
    lazy = false,
    version = false, -- always use the latest git commit
  },
  git = {
    cooldown = 60,
  },
  install = { colorscheme = { "catppuccin", "tokyonight" } },
  checker = { enabled = false }, -- automatically check for plugin updates
  performance = {
    rtp = {
      -- disable some rtp plugins
      disabled_plugins = {
        "gzip",
        "tarPlugin",
        "tohtml",
        "tutor",
        "zipPlugin",
      },
    },
  },
})
```

**Comment by iamxiaojianzheng** [comment]
![image](https://github.com/user-attachments/assets/e6e59e72-610b-40cc-bf57-22485851bc71)

**Comment by folke** [comment]
That's not how you configure using main. Please check the docs website for more info

**Reply by iamxiaojianzheng** [reply]
I didn't manually configure LazyVim, which defaults to branch as main. 

Then every time I update, It just update to the latest tag, not commit.

**Reply by folke** [reply]
It does not. It defaults to `version = "*"`

See here: https://www.lazyvim.org/configuration/lazy.nvim

**Reply by iamxiaojianzheng** [reply]
I have manually configured 'version = false', which means update to the latest commit, right?

**Reply by folke** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/9a76832d-6506-4166-afd1-7dfedc809fb4)

**Reply by iamxiaojianzheng** [reply]
Oh, oh, I misunderstood.
